CONTRIBUTING.md

```diff
@@ -0,0 +1,164 @@
+<!-- omit in toc -->
```

Comment:
Is there a way to test running a TOC on a branch if possible before merging? Minor point.

It seems like a cool feature.

https://markdown-all-in-one.github.io/docs/guide/table-of-contents.html#overview